rules/rules.libsonnet

```diff
@@ -358,6 +358,34 @@
               )
             ||| % $._config,
           },
+          {
+            record: 'node:disk_inodes_total:',
+            expr: |||
+              max(
+                max(
+                  kube_pod_info{%(kubeStateMetricsSelector)s, host_ip!=""}
+                ) by (node, host_ip)
+                * on (host_ip) group_right (node)
+                label_replace(
+                  (max(node_filesystem_files{mountpoint="/"}) by (instance)), "host_ip", "$1", "instance", "(.*):.*"
+                )
+              ) by (node)
+            ||| % $._config,
+          },
+          {
+              record: 'node:disk_inodes_free:',
+              expr: |||
+                max(
+                  max(
+                    kube_pod_info{%(kubeStateMetricsSelector)s, host_ip!=""}
+                  ) by (node, host_ip)
+                  * on (host_ip) group_right (node)
+                  label_replace(
+                    (max(node_filesystem_files_free{mountpoint="/"}) by (instance)), "host_ip", "$1", "instance", "(.*):.*"
+                  )
```

Comment:
Same as above.

Reply:
I've push a new commit for this PR to address the concerns you raised.
Let me know if it's ok or not

Thanks
Ben

Reply:
Is there anything else I should change ?

Reply:
No, I just need some time to test the dashboard. :wink: :relaxed: Sorry.